lib/Crypto/doc/sailfish-crypto.qdocconf

```diff
@@ -36,13 +36,18 @@ qhp.SailfishCrypto.subprojects.classes.indexTitle = Sailfish Crypto C++ Classes
 qhp.SailfishCrypto.subprojects.classes.selectors = class fake:headerfile
 qhp.SailfishCrypto.subprojects.classes.sortPages = true
 
+qhp.SailfishSecrets.subprojects.classes.title = QML Types
+qhp.SailfishSecrets.subprojects.classes.indexTitle = Sailfish.Crypto QML Types
+qhp.SailfishSecrets.subprojects.classes.selectors = qmlclass
+qhp.SailfishSecrets.subprojects.classes.sortPages = true
+
 qhp.SailfishCrypto.filterAttributes = sailfish-crypto 1.0.0
 qhp.SailfishCrypto.customFilters.SailfishCrypto.name = Sailfish OS Crypto Library 0.0.1
 qhp.SailfishCrypto.customFilters.SailfishCrypto.filterAttributes = sailfish-crypto 0.0.1
 
 HTML.footer += \
     "<div class=\"footer\">\n" \
-    "  <p><acronym title=\"Copyright\">&copy;</acronym> 2018 Jolla Ltd.</p>\n" \
+    "  <p><acronym title=\"Copyright\">&copy;</acronym> 2018-2022 Jolla Ltd.</p>\n" \
```

Comment:
```suggestion
    "  <p><acronym title=\"Copyright\">&copy;</acronym> 2018 - 2022 Jolla Ltd.</p>\n" \
```
Nitpick: in the style we have space around the dash. Feel free to ignore this one.